src/components/Scatterplot/index.js

```diff
@@ -148,7 +150,19 @@ const ScatterplotComponent = ({
     <Layout
       chart={
         <svg style={{ width: '100%', height: '100%' }}>
-          <GridLines grid={grid} {...chartSize} />
+          {React.Children.map(children, child => {
+            const childProps = {
+              ...chartSize,
+              axes: {
+                ...(child.props || {}).axes,
+                [Axes.x]:
+                  ((child.props || {}).axes || {}).x === undefined
+                    ? String(Axes.x)
+                    : child.props.axes.x,
+              },
+            };
+            return React.cloneElement(child, childProps);
+          })}
```

Comment:
Probably not now, but I would think about having helper for this type of things for `LineChart` and `Scatterplot`

Reply:
Yeah it will likely be extracted as we get more and more components pulled out like this.

Or maybe they'll diverge and we won't want them consolidated -- time will tell.